src/user/announcements.rst

```diff
@@ -8,6 +8,11 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2021
 ----
 
+:2021-09-30: ``defaults`` channel is now dropped when building conda-forge packages
```

Comment:
Should we say something about the corner cases where it may be needed and/or how to avoid them?

Reply:
I don't know any corner cases right now